README.md

```diff
@@ -179,6 +179,17 @@ python data_list/get_meta_file.py
 sh scripts/train.sh
 ```
 
+## ONNX export
+Support for ONNX model exports is now available
+```
+python export.py
+usage: export.py [-h] [--input_size INPUT_SIZE] [--batch_size BATCH_SIZE] --model_path MODEL_PATH [--model_size MODEL_SIZE] 
+[--decoder_type DECODER_TYPE] [--export_path EXPORT_PATH] [--opset OPSET]
+export.py: error: the following arguments are required: --model_path
```

Comment:
This line(L188) should be deleted.

Reply:
@piddnad just L188 or L185 ... L188?

Reply:
Only L188. It seems to be an accidentally copied error log?
("export.py: error: the following arguments are required: --model_path")